### PR TITLE
Check completed - nextflow comparison with main using hg38

### PR DESCRIPTION
**Pull request by hyunhwan-bcm**
This version has been modified version of `nextflow_conversion` to match the output of the [`main_fix_sort`](https://github.com/hyunhwan-bcm/AI_MARRVEL/tree/main_fix_sort) branch. Both branches now use stable sorting for almost 100% of its sorts. Additionally, the comparisons are identical after sorting the integrated output, with a precision error margin of 1e-9.

There are a couple of things to note:

1. The `add_n_c.py` script still has the issue of not merging the clin_var profile, but we can fix it once the other fixes produce the same result as `main_fix_sort`.
2. The `phrank` input needs to be changed - the current version receives an unfiltered VCF.
3. The `generate_*.py` scripts were reverted, and we need to ensure that the versions in `nextflow_conversion` can generate identical files.

### CODE REVIEW

**Review by hyunhwan-bcm — COMMENTED** [review]
finally merged

File: bin/add_c_nc.py [hunk 1]
```diff
@@ -26,9 +26,6 @@ def add_c_nc(score, ref):
     bl = clin_nc.new_start.values
     bc = clin_nc.new_chr.values
 
-    bc[bc=='MT'] = -1 # Treat MT chromosome as NaN values to avoid string comparison
-    bc = bc.astype(int)
```
Comment (jylee-bcm) [inline bin/add_c_nc.py:30]: I thought having this code is more accurate. Do you think it's better to revert this change?

**Review by jylee-bcm — APPROVED** [review]
